src/hokusai/hokusai.yml

```diff
@@ -26,7 +26,8 @@ commands:
   setup-docker:
     steps:
       - setup
-      - setup_remote_docker
+      - setup_remote_docker:
+        version: 19.03.13
```

Comment:
```suggestion
```

This isn't what that message meant! At the top of the file there's a comment that governs the orb version. That comment needs to be bumped.

Reply:
@pepopowitz this line does need to be removed.

Reply:
If this line needs to be removed, there's no need for this PR. This is the only significant change in this PR - the body of the PR describes why I introduced it.

If it is the case that this won't resolve the permission error in the linked build, can you add context as to what is needed instead & why?

Reply:
OH! Yeah, of course. Sorry, I'm having a moment, ha.